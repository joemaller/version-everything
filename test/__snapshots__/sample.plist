<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
  <dict>
    <key>Animals</key>
    <array>
      <string>dog</string>
      <string>cat</string>
    </array>
    <key>Label</key>
    <string>Sample plist file</string>
    <key>Version</key>
    <string>1.414.213</string>
  </dict>
</plist>
