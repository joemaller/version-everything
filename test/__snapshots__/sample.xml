<?xml version="1.0"?>
<catalog>
  <version>1.414.213</version>
  <!-- From https://docs.microsoft.com/en-us/previous-versions/windows/desktop/ms762271(v=vs.85) -->
  <book id="bk101">
    <author>Joe</author>
    <title>XML Test File</title>
  </book>
</catalog>